rusk-wallet/README.md

```diff
@@ -7,23 +7,42 @@ USAGE:
     rusk-wallet [OPTIONS] [SUBCOMMAND]
 
 OPTIONS:
-    -d, --data-dir <DATA_DIR>          Directory to store user data [default: /Users/$(whoami)/.dusk]
-    -n, --wallet-name <NAME>           Name for your wallet [default: $(whoami).dat]
-    -f, --wallet-file <PATH>           Path to a wallet file. Overrides `data-dir` and `wallet-
-                                       name`, useful when loading a wallet that's not in the default
-                                       directory
-    -a, --rusk-addr <RUSK_ADDR>        Rusk address [default: 127.0.0.1]
-    -p, --rusk-port <RUSK_PORT>        Rusk port [default: 8585]
-        --prover-addr <PROVER_ADDR>    Prover service address [default: `rusk_addr`]
-        --prover-port <PROVER_PORT>    Prover service port [default: `rusk_port`]
-    -i, --ipc-method <IPC_METHOD>      IPC method for communication with rusk [uds, tcp_ip]
-                                       [default: uds]
-    -s, --socket-path <SOCKET_PATH>    Path for setting up the unix domain socket [default: /tmp/
-                                       rusk_listener]
-        --skip-recovery                Skip wallet recovery phrase (useful for headless wallet
-                                       creation)
-    -h, --help                         Print help information
-    -V, --version                      Print version information
+    -d, --data-dir <DATA_DIR>
+            Directory to store user data [default: `$HOME/.dusk`]
+
+    -n, --wallet-name <NAME>
+            Name for your wallet [default: `$(whoami)`]
+
+    -f, --wallet-file <PATH>
+            Path to a wallet file. Overrides `data-dir` and `wallet-name`,
+            useful when loading a wallet that's not in the default directory
+
+    -a, --rusk-addr <RUSK_ADDR>
+            Rusk address
+
+    -p, --rusk-port <RUSK_PORT>
+            Rusk port
+
+        --prover-addr <PROVER_ADDR>
+            Prover service address [default: `rusk_addr`]
+
+        --prover-port <PROVER_PORT>
+            Prover service port [default: `rusk_port`]
+
+    -i, --ipc-method <IPC_METHOD>
+            IPC method for communication with rusk [uds, tcp_ip]
```

Comment:
Its unclear what is the bind address if the user picks `tcp_ip`. It is implicit that we are talking about `rusk-addr`.

I suggest we take a single argument `rusk-socket` that accepts either UDS sockets or TCP/IP addresses (the program will attempt to parse it). That way, we greatly reduce complexity.

Reply:
This is indeed a great idea